Base/usr/share/man/man6/BrickGame.md

```diff
@@ -16,7 +16,7 @@ BrickGame is a classic game. Pieces consisting of four small squares each fall f
 
 You can control where and how a piece falls, by moving it left or right, making it move down faster, dropping it down instantly, or rotating it left and right. There are multiple control schemes available for these six basic actions; and the `space bar` always serves as the instant drop key.
 
--   `Left`: Move left, `Right`: Move right, `Up`: Rotate right, `Down`: Move down
+-   `Left`: Move left, `Right`: Move right, `Up`: Rotate right, `Z`: Rotate left, `Down`: Move down
```

Comment:
Instead of inserting this in the middle of the arrow keys, maybe add it to the end of the last row, then it happens to be below the other "Rotate left" which is vaguely pleasing.

Reply:
I added it to the first row because I see each row as a single cohesive unit of key bindings that can be used to play the game.

Reply:
> ps: Consider doing a self-review before publishing a PR and try to catch things like this yourself :)

I do it every time.

Reply:
Done.